src/csharp/experimental/README.md

```diff
@@ -17,6 +17,19 @@ Xamarin.iOS
 - supported architectures: arm64 (iPhone 6+) and x86_64 (iPhone simulator)
 
 # Unity
-gRPC C# currently doesn't support Unity, but some proof-of-concept
-work has been done. There is in-progress effort to provide users
-with a pre-built gRPC package that can be used in their projects.
+
+gRPC C# now has experimental support for Unity. Please try using gRPC with
+Unity and provide feedback!
```

Comment:
Maybe add where to do this feedback ?

Reply:
I added this mostly to encourage people to file an issue if something doesn't work for them and that part doesn't require additional explanation IMHO.

Btw, I plan to add a unity helloworld example, so this text will probably change relatively soon anyway.